tests/lsdb/catalog/test_nested.py

```diff
@@ -0,0 +1,15 @@
+import nested_dask as nd
+import pandas as pd
+
+from lsdb import Catalog
+
+
+def test_dropna(small_sky_with_nested_sources):
+    filtered_cat = small_sky_with_nested_sources.query("sources.mag < 15.1")
+    drop_na_cat = filtered_cat.dropna()
```

Comment:
It might be good to also add a test that checks the unique nested-dask dropna behavior, so something that would use the subset or on_nested kwargs